### PR TITLE
Update copy

### PR DESCRIPTION
**Pull request by jontg**
Needs reflow before we merge!

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -88,9 +88,9 @@
     <div class="row">
         <div class="col-md-6">
             <div class="intro">
-                <p>Getting involved in politics doesn’t have to be complicated or confusing. When you join <em>Calls for Change</em>, we'll automatically pair you with your members of Congress and send you a text or email reminder to call them once a week. We’ll also send you information on a suggested topic that we think is important to women and equality more broadly.</p>
+                <p>It's easier than you think to do something to advance women's rights and quality broadly (<a herf="#about">more details</a>).  Calling members of Congress is the most effective way to influence the political process, and Calls for Change makes it <a href="how-it-works">simple and fast</a>.  Each week we'll send you all the infromation you need — from the right phone number to a sample script — so in just ten minutes you can make sure your voice is heard.</p>
```
Comment (sethvoltz) [inline static/index.html]: Do we have have a `how-it-works` page or is this supposed to be `#how-it-works`?

File: static/index.html [hunk 2]
```diff
@@ -159,25 +159,20 @@
...
         <div class="col-md-9 col-md-offset-3">
-            <section class="how-it-works">
+            <section id="how-it-works" class="how-it-works">
                 <h3>How it Works</h3>
                 <ul>
-                    <li>Enter your information on the form on this page. (It only takes 30 seconds!)</li>
-                    <li>Each week, we’ll text or email you talking points on a timely issue and your representatives’ phone numbers for one-tap calling from your smartphone. </li>
-                    <li>That’s it! When you make your calls, you make change happen.</li>
+                    <li>Each week we text and/or email you background info on a timely topic, a sample script, and the phone numbers for your Congress members for one-tap calling from your smartphone.</li>
+                    <li>You make a few quick calls, and your opinion is logged and passed on to your Congress members. (<a href="http://theslot.jezebel.com/how-to-effectively-lobby-your-congressperson-1788958124">Learn more about why calling works</a>)</li>
```
Comment (sethvoltz) [inline static/index.html]: Make this link open a new window/tab: `target="_blank"`

**Review by sethvoltz — CHANGES_REQUESTED** [review]

File: static/index.html (hunk 1 above)
Comment (blake213) [reply static/index.html]: Code says `<a herf="#about">more details</a>` (herf instead of href)

Comment (blake213) [reply static/index.html]: `and quality broadly` should that be `equality`?

File: static/index.html (hunk 2 above)
Comment (jontg) [reply static/index.html]: fixed.

File: static/index.html (hunk 1 above)
Comment (jontg) [reply static/index.html]: Fixed, fixed and fixed.

**Comment by jontg** [comment]
@blake213 can you take a look at how this renders locally (you can run `bin/serve` then navigate to `http://127.0.0.1:8080` to see how it lays out)?

**Review by sethvoltz — APPROVED** [review]

**Pull request merged**